### ISSUE TITLE
How Rebuild the solution?

### ISSUE THREAD

**Issue by dropyghost** [Esri/route-planner-csharp#26]
I follow the requirement on the main page

- ArcGIS Runtime SDK for WPF
- Xceed Data Grid for WPF (www.xceed.com).
- ActiveReports for .NET 3.0 (www.datadynamics.com).

But still getting some errors.

![image](https://cloud.githubusercontent.com/assets/10077853/13640701/f0319e3a-e5e7-11e5-9207-e63693b850f9.png)

What are the proper step to Rebuld the Solution. I just want change the app to use my road network.

**Comment by rConger** [comment]
You don’t need to rebuild the app to use your own road network. Just use your network to publish route and VRP services and point to your services in services.xml.

-Rodney

**Comment by dropyghost** [comment]
I install the prebuild version and doesnt have a services.xml file there.
Also beside road network can I change setting to also use my maps?

> C:\Program Files (x86)\RoutePlanner

![image](https://cloud.githubusercontent.com/assets/10077853/13641767/df00cf74-e5eb-11e5-9f0f-266ea5b8de09.png)

**Comment by rConger** [comment]
Yes, all of this is in services.xml. C:\Program Data\RoutePlanner\services.xml

BTW, Program Data is a hidden directory by default.

**Comment by dropyghost** [comment]
I change the services.xml and try replace the street satellite map, but didnt see any change or error, and still show same maps like is reading it from somewhere else.

```
    <service type="ArcGISCached" server="server.arcgisonline.com" basemap="true" visible="true">
      <name>Streets</name>
      <title>StreetMap World 2D</title>
      <soapurl>https://server.arcgisonline.com/arcgis/services/World_Street_Map/MapServer</soapurl>
      <opacity>1</opacity>
    </service>
    <service type="ArcGISCached" server="server.arcgisonline.com" basemap="true" visible="false">
      <name>Satellite</name>
      <title>Imagery World 2D</title>      
      <soapurl>https://server.arcgisonline.com/arcgis/services/World_Imagery/MapServer</soapurl>
      <opacity>1</opacity>
    </service>
```

Also looks like my map is a different format/type as the one is beign used by planner.

My Map

![image](https://cloud.githubusercontent.com/assets/10077853/14187769/5feaddd2-f74e-11e5-958c-6a143dbcc59a.png)

ArcGis 

![image](https://cloud.githubusercontent.com/assets/10077853/14187781/6ea50eb0-f74e-11e5-82d7-4747da33c6bf.png)

So how I know if Im replacing the map service correctly and what format should I publish my map?
